python/sglang/srt/managers/router/model_rpc.py

```diff
@@ -558,9 +557,7 @@ def handle_finished_requests(self, batch: Batch):
 
                 meta_info = {
                     "prompt_tokens": req.prompt_tokens,
-                    "completion_tokens": len(req.input_ids)
-                    + len(req.output_ids)
-                    - req.prompt_tokens,
+                    "completion_tokens": len(req.output_ids),
```

Comment:
please revert this. see reasons here
https://github.com/sgl-project/sglang/blob/9472363a3b20dd7ddefe1ccf0d185de9fef5e980/python/sglang/srt/managers/router/infer_batch.py#L30-L32

Reply:
reverted. moving forward, it maybe good to keep req object immutable and clone req for internal processing, otherwise, if we actually need the original req.input_ids, we have to defensive copy into req itself.